editoast/src/views/mod.rs

```diff
@@ -127,19 +132,29 @@ async fn version() -> Json<Version> {
     })
 }
 
+#[get("/version/core")]
+async fn core_version(core: Data<CoreClient>) -> Json<Version> {
+    let mut response = CoreVersionRequest {}.fetch(&core).await.unwrap();
+    if response.git_describe.is_some() {
+        response.git_describe = None;
```

Comment:
What’s the point to always return `None` ?

Reply:
In local mode, you are always obliged to return `None`. Locally, we don't set the OSRD_GIT_DESCRIBE envvar!